### DISCUSSION TITLE
SkyWalking agent version 8.8.0 is unable to generate TIDs (trace IDs) for RocketMQ 5.0

### DISCUSSION THREAD

**Discussion by JackSlamDunk** [apache/skywalking#11006] [General]
![image](https://github.com/apache/skywalking/assets/58261759/cbf5fdf9-eb6c-4cdb-99fe-e104d2f854c9)

**Comment by wu-sheng** [comment]
We never guarantee you have trace ID for every log. That is misunderstanding.

**Reply by JackSlamDunk** [reply]
However, RocketMQ4.x can generate tid.

**Reply by wu-sheng** [reply]
This is controlled by plugin. 
You could check whether your using version is support by that agent version 
Plugins are contributed by global contributors, we don't control or manage them, we just host them.
Reach rocketmq community maybe someone(only maybe) knows this.

**Reply by wu-sheng** [reply]
There is no however. You are thinking wrong.

**Reply by JackSlamDunk** [reply]
ok Thank you for your answer

**Comment by JackSlamDunk** [comment]
Hi skywalking-java already supports RocketMQ5.0 plug-in on the main branch, which is supported in version 9.0, but there is no version 9.0 Agent download on the skywalking official website, I would like to consult when skywalking will release 9.0 Agent

**Reply by wu-sheng** [reply]
We wouldn't release shortly. We release in 2-3 monthly periodically.

**Reply by JackSlamDunk** [reply]
OK  Thanks
